### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by HossainAraf**
**Method implemented:**

- Create a table named owners with the following columns:
   id: integer (set it as autoincremented PRIMARY KEY), full_name: string, age: integer
- Create a table named species with the following columns:
   id: integer (set it as autoincremented PRIMARY KEY)
   name: string
- Modify animals table:
  Make sure that id is set as autoincremented PRIMARY KEY
- Remove column species
- Add column species_id which is a foreign key referencing species table
- Add column owner_id which is a foreign key referencing the owners table
- Remember all this goes in schema.sql file.
- Insert the provided data into the owners table:
- Insert the following data into the species table:  Pokemon, Digimon
- Modify your inserted animals so it includes the species_id value:
  If the name ends in "mon" it will be Digimon
 All other animals are Pokemon
- Modify your inserted animals to include owner information (owner_id):
- Remember these insertions and modifications happen in data.sql file.
- Write queries (using JOIN) to answer the following questions:
  What animals belong to Melody Pond?
  List of all animals that are pokemon (their type is Pokemon).
  List all owners and their animals, remember to include those that don't own any animal.
  How many animals are there per species?
  List all Digimon owned by Jennifer Orwell.
  List all animals owned by Dean Winchester that haven't tried to escape.
  Who owns the most animals?

- Remember all these should be written in queries.sql file.

### SCREENSHOTS
Queries-12Dec (1)
Queries-12Dec (2)
TRansation (1)
TRansation (2)
TRansation (3)
TRansation (4)
TRansation (5)
TRansation (6)

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -14,4 +14,38 @@ VALUES
...
     (10, 'Blossom', '1998-10-13', 3, true, 17),
-    (11, 'Ditto', '2022-05-14', 4, true, 22);
\ No newline at end of file
+    (11, 'Ditto', '2022-05-14', 4, true, 22);
+
+    --INSERT full-name and age into owners table
+INSERT INTO owners (full_name, age)
+VALUES
+    ('Sam Smith', 34),
+    ('ennifer Orwell', 19),
```
Comment (zieeco) [inline data.sql]: - Hey, kidly fix the misspelled name (Jennifer Orwell)

File: queries.sql [hunk 2]
```diff
@@ -98,4 +98,14 @@ FROM animals
...
+SELECT a.name
+FROM animals AS a
+JOIN owners AS o ON a.owner_id = o.id
+WHERE o.full_name = 'Melodie Pond';
+
 --
+SELECT animals.name
+FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
```
Comment (zieeco) [inline queries.sql]: - Hey, you've only added two queries, Please, kindly add the remaning queries as required from the project requirement.


image

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @HossainAraf,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Nice query on the multiple tables.✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions

- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (HossainAraf) [reply queries.sql]: @zieeco Thanks for the feedback.

**Review by topeogunleye — APPROVED** [review]
### Hi @HossainAraf, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Added all the remaining queries. ✔️ 
 
## [Optional] suggestions:
* [ ]  Please kindly add the screenshots results of your queries.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**